### PR TITLE
New Brunswick Pension Benefit

### LINKED ISSUE #280: New Brunswick Pension Benefit
Reference: https://www2.gnb.ca/content/gnb/en/services/services_renderer.2215.Pension_Benefits.html

### PR DESCRIPTION
**Pull request by YiweiJ**
Fixes #280

### CODE REVIEW

File: policyengine_canada/variables/gov/provinces/nb/benefits/pension_benefit/nb_pension_benefit.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from policyengine_canada.model_api import *
+
+
+class nb_pension_benefit(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/nb/benefits/pension_benefit/nb_pension_benefit.py:4]: make a nb_pension_benefit_eligible.py

File: policyengine_canada/variables/gov/provinces/nb/benefits/pension_benefit/nb_pension_benefit.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        age = person("age", period)
+        pension_income = person("pension_and_savings_plan_income", period)
+        threshold = parameters(
+            period
+        ).gov.provinces.nb.benefits.pension_benefit.age_eligibility
+        maximum_benefit = parameters(
+            period
+        ).gov.provinces.nb.benefits.pension_benefit.maximum_benefit
+        age_eligibility = age > threshold
+        # 1_000 or less
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/nb/benefits/pension_benefit/nb_pension_benefit.py:22]:
```suggestion
        # Capped at a certain amount
```

File: policyengine_canada/parameters/gov/provinces/nb/benefits/pension_benefit/maximum_benefit.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+description: New Brunswick provides this maximum pension benefit.
+values:
+  2022-01-01: 1_000
+metadata:
+  unit: currency-CAD
+  label: New Brunswick Maximum Pension Benefit
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/nb/benefits/pension_benefit/maximum_benefit.yaml:6]:
```suggestion
  label: New Brunswick pension benefit max amount
```

File: policyengine_canada/parameters/gov/provinces/nb/benefits/pension_benefit/maximum_benefit.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/nb/benefits/pension_benefit/maximum_benefit.yaml:1]: rename this to max_amount

File: policyengine_canada/parameters/gov/provinces/nb/benefits/pension_benefit/age_eligibility.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/nb/benefits/pension_benefit/age_eligibility.yaml:1]: delete this

File: changelog_entry.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - New Brunswick Pension Benefit.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - New Brunswick pension benefit. 
```

File: policyengine_canada/tests/gov/provinces/nb/pension_benefits/nb_pension_benefit.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/tests/gov/provinces/nb/pension_benefits/nb_pension_benefit.yaml:1]: adj test

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_canada/parameters/gov/provinces/nb/benefits/pension_benefit/max_amount.yaml
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nb/benefits/pension_benefit/max_amount.yaml:1]: please rename cap.yaml

**Review by MaxGhenis — APPROVED** [review]
good to merge after this

**Pull request merged**